test-network/scripts/deployCC.sh

```diff
@@ -33,54 +33,31 @@ CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:])
-  if [ "$CC_NAME" = "basic" ]; then
-    println $'\e[0;32m'asset-transfer-basic$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-basic"
-  elif [ "$CC_NAME" = "events" ]; then
-    println $'\e[0;32m'asset-transfer-events$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-events"
-  elif [ "$CC_NAME" = "secured" ]; then
-    println $'\e[0;32m'asset-transfer-secured-agreeement$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-secured-agreement"
-  elif [ "$CC_NAME" = "ledger" ]; then
-    println $'\e[0;32m'asset-transfer-ledger-agreeement$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-ledger-queries"
-  elif [ "$CC_NAME" = "private" ]; then
-    println $'\e[0;32m'asset-transfer-private-data$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-private-data"
-  elif [ "$CC_NAME" = "sbe" ]; then
-    println $'\e[0;32m'asset-transfer-sbe$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-sbe"
-  else
-    fatalln "The chaincode name ${CC_NAME} is not supported by this script. Supported chaincode names are: basic, events, ledger, private, sbe, secured"
-  fi
+#User has not provided a name
+if [ -z "$CC_NAME" ]; then
+  fatalln "No chaincode name was provided. Valid call example: ./network.sh deployCC -ccn \"basic\" -ccp \"../asset-transfer-basic/chaincode-go\" -ccl \"go\""
+fi
 
-  # now see what language it is written in
-  if [ "$CC_SRC_LANGUAGE" = "go" ]; then
-    CC_SRC_PATH="$CC_SRC_PATH/chaincode-go/"
-  elif [ "$CC_SRC_LANGUAGE" = "java" ]; then
-    CC_SRC_PATH="$CC_SRC_PATH/chaincode-java/"
-  elif [ "$CC_SRC_LANGUAGE" = "javascript" ]; then
-    CC_SRC_PATH="$CC_SRC_PATH/chaincode-javascript/"
-  elif [ "$CC_SRC_LANGUAGE" = "typescript" ]; then
-    CC_SRC_PATH="$CC_SRC_PATH/chaincode-typescript/"
-  fi
+# User has not provided a path
+if [ -z "$CC_SRC_PATH" ]; then
+  fatalln "No chaincode path was provided. Valid call example: ./network.sh deployCC -ccn \"basic\" -ccp \"../asset-transfer-basic/chaincode-go\" -ccl \"go\""
+fi
 
-  # check that the language is available for the sample chaincode
-  if [ ! -d "$CC_SRC_PATH" ]; then
-    fatalln "The smart contract language \"$CC_SRC_LANGUAGE\" is not yet available for the \"$CC_NAME\" sample smart contract"
-  fi
-## Make sure that the path the chaincode exists if provided
-elif [ ! -d "$CC_SRC_PATH" ]; then
+## Make sure that the path to the chaincode exists
+if [ ! -d "$CC_SRC_PATH" ]; then
   fatalln "Path to chaincode does not exist. Please provide different path"
 fi
 
+# User has not provided a language
+if [ -z "$CC_SRC_LANGUAGE" ]; then
+  fatalln "No chaincode language was provided. Valid call example: ./network.sh deployCC -ccn \"basic\" -ccp \"../asset-transfer-basic/chaincode-go\" -ccl \"go\""
+fi
+
+# check that the language is available for the sample chaincode
+if [ ! -d "$CC_SRC_PATH" ]; then
+  fatalln "The smart contract language \"$CC_SRC_LANGUAGE\" is not yet available for the \"$CC_NAME\" sample smart contract"
```

Comment:
```
if [ ! -d "$CC_SRC_PATH" ]; then
  fatalln "The smart contract language \"$CC_SRC_LANGUAGE\" is not yet available for the \"$CC_NAME\" sample smart contract"
```

This is duplicative with the if statement above and should be removed. The statement that the language is not available is only relevant for the asset-trasnfer samples that could be called with short names.